src/components/FormModal/FormModal.js

```diff
@@ -62,10 +68,59 @@ export default function FormModal({ onCancelAdding, id, name }) {
     setOpeningDate(date);
   }
 
+  const routineData = getDataFromLocalStorage();
+
+  const sameProduct = routineData.filter((sameProduct) => {
+    return sameProduct.id === id;
+  });
+
+  function sendNameToIndex(name) {
+    const i = getIndexForWeekday(name);
+    setIndexWeekday(i);
+  }
+
+  const sameProductMorning = sameProduct.filter((productObject) => {
+    const morning = productObject.days[indexWeekday].morning === true;
+    return morning;
+  });
+
+  const sameProductEvening = sameProduct.filter((productObject) => {
+    const evening = productObject.days[indexWeekday].evening === true;
+    return evening;
+  });
+
   function handleModalFormSubmit(event) {
     event.preventDefault();
-    sendDataToLocalStorage(weekRoutine);
-    onCancelAdding();
+    if (tickedTwice === false) {
+      sendDataToLocalStorage(weekRoutine);
+      onCancelAdding();
+    } else {
+      alert(
+        "Oops, you are trying to add the product to a time when you are already using it, please untick before adding"
+      );
+    }
+  }
+
+  function checkMorningAddedProducts(name, morningClicked) {
+    if (sameProductMorning.length !== 0 && !morningClicked) {
+      if (name === sameProductMorning[0].days[indexWeekday].name) {
```

Comment:
Could you combine these two condition into one?

Reply:
Can I? Because in the "if" above, I am already combining two conditions. Can I add a third?

Reply:
Yes you can. Look at some `crazy` conditions I made in my capstone:

```
 function validationPassedWithEvent() {
    return !(
      !sceneSelect ||
      isEmptyScene ||
      isEmptyEvent ||
      hasOnlyZeros ||
      lowerThirdInHasOnlyZeros ||
      lowerThirdOutHasOnlyZeros ||
      !isCorrectLowerThirdIn ||
      !isCorrectLowerThirdLength ||
      !checkCorrectTimeCode()
    )
  }
```

Reply:
😂 WOW
I don't know why, but i was convinced that you can only give two conditions 🙈 
This will make life a lot easier